examples/README.md

```diff
@@ -420,6 +421,18 @@ examples][intel-examples-readme].
 
 [daos-slurm.yaml]: ../community/examples/intel/daos-slurm.yaml
 
+### [hpc-cluster-amd-slurmv5.yaml] ![community-badge]
+
+This example provisions a Slurm cluster using AMD VM machine types. It
+automates the initial setup of Spack, including a script that can be used to
+install the AMD Optimizing C/C++ Compiler ([AOCC]) and compile OpenMPI with
+AOCC. It is more extensively in a dedicated [README for AMD
```

Comment:
A word got dropped I think:
It is **documented** more extensively...

Reply:
Done on HEAD